app/views/admin/worldwide_organisations_translations/index.html.erb

```diff
@@ -1,55 +1,82 @@
-<% page_title @worldwide_organisation.name + " translations" %>
+<% content_for :page_title, "#{@worldwide_organisation.name} translations" %>
+<% content_for :title, @worldwide_organisation.name %>
+<% content_for :context, "Organisation" %>
+<% content_for :title_margin_bottom, 4 %>
 
-<%= content_tag_for(:div, @worldwide_organisation) do %>
-  <div class="organisation-header">
-    <h1><%= @worldwide_organisation.name %></h1>
-    <%= view_on_website_link_for @worldwide_organisation %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body"><%= view_on_website_link_for @worldwide_organisation, class: "govuk-link" %></p>
   </div>
+</div>
 
-  <section class="organisation-details">
-    <%= tab_navigation_for(@worldwide_organisation) do %>
-      <h2>Translations</h2>
+<%= render "components/secondary_navigation", {
+  aria_label: "Worldwide organisation navigation tabs",
+  items: secondary_navigation_tabs_items(@worldwide_organisation, request.path)
+} %>
 
-      <% if @worldwide_organisation.non_english_translated_locales.present? %>
-        <table id="worldwide-organisations-translations" class="table table-striped table-bordered">
-          <thead>
-            <tr class="table-header">
-              <th>Locale</th>
-              <th>Actions</th>
-            </tr>
-          </thead>
-          <tbody>
-            <% @worldwide_organisation.non_english_translated_locales.each do |locale| %>
-              <tr>
-                <td class="locale">
-                  <%= link_to locale.native_language_name, edit_admin_worldwide_organisation_translation_path(@worldwide_organisation, locale.code) %> (<%= link_to "view", @worldwide_organisation.public_path(locale: locale.code) %>)
-                </td>
-                <td class="actions">
-                  <%= button_to 'Delete',
-                        admin_worldwide_organisation_translation_path(@worldwide_organisation, locale.code),
-                        method: :delete,
-                        class: "btn btn-danger",
-                        data: { confirm: "Are you sure you want to delete this translation?" } %>
-                </td>
-              </tr>
-            <% end %>
-          </tbody>
-        </table>
-      <% else %>
-        <p class="no-content no-content-bordered">No translations</p>
-      <% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Translations",
```

Comment:
This is part of the overall design issues responded to elsewhere. Nik’s preference is to leave it as it is in this instance until it can be addressed in his newer design.

Reply:
Further to this: Nik has decided to revert this to "Locale". I think a hading is more suitable here since semantically it relates to the section as a whole rather than a column in the table that would be the more appropriate use of the table head element.